### PR TITLE
Feature/cache rating field

### PR DESCRIPTION
**Pull request by Yamsha75**

### CODE REVIEW

File: api/everycache_api/api/schemas/cache.py [hunk 1]
```diff
@@ -22,6 +22,13 @@ class CacheSchema(ma.SQLAlchemyAutoSchema):
     description = ma.String(validate=validate.Length(min=5), required=False)
     rating = ma.Float(dump_only=True)
 
+    @post_dump
```
Comment (Nieminik) [inline api/everycache_api/api/schemas/cache.py:25]: I know there is not much to do about duplicating this one, but we could at least get the common code part and put it in a function. Then call it here and in the PublicCacheSchema post_dump method.

File: api/everycache_api/common/filters.py [hunk 2]
```diff
@@ -1,79 +1,70 @@
...
-def apply_query_filters(query, schema):
-    args = request.args.to_dict(flat=False)
+            if not operation:
+                continue
 
-    for key, values_list in args.items():
-        if key not in schema.fields:
-            continue
+            try:
+                schema_field.deserialize(value)
```
Comment (Nieminik) [inline api/everycache_api/common/filters.py:64]: Big kudos

**Review by Nieminik — APPROVED** [review]

File: api/everycache_api/api/schemas/cache.py (hunk 1 above)
Comment (Yamsha75) [reply api/everycache_api/api/schemas/cache.py:25]: Yes, we should have tried reusing code in similar schemas, but we have limited time and cannot afford to refactor this

Comment (Yamsha75) [reply api/everycache_api/api/schemas/cache.py:25]: I've actually been able to do it in #40

**Pull request merged**